README.md

```diff
@@ -14,47 +23,123 @@ And then execute:
 
     $ bundle
 
-Or install it yourself as:
+Or execute the following line to install the gem without bundler.
 
     $ gem install ricohapi-mstorage
 
 ## Usage
 
 You can find [a working rails sample app using Ricoh Media Storage API](https://github.com/ricohapi/media-storage-sample-app).
 
-### Obtaining Access Token for Media Storage API
+## SDK API
+Require `ricohapi/mstorage` to use the APIs below.
```

Comment:
I think the following explanation is better: "Add `require 'ricohapi/mstorage'` to use the APIs below."
Because end user can copy it.